openpype/pipeline/farm/pyblish_functions.py

```diff
@@ -583,7 +583,7 @@ def _create_instances_for_aov(instance, skeleton, aov_filter, additional_data,
         else:
             # in case of single frame
             cam = [c for c in cameras if c in col]
-        if cam:
+        if cam or subset != "maxrenderMain":
```

Comment:
What's your camera name? :) What's your `col` value?
So what's in `instance.data["cameras"]`?

Reply:
It's mainly because the max hosts act differently in terms of multiple camera submissions. It will create mutliple scene to submit deadline for multiple camera submissions.(Unlike maya). And the subset name would follow the way how I set in the files(maxRenderMain_Camera001) so if I dont adjust this, it will duplicate the camera name and the camera name become a list. 
But it is good remainder, and I will introduce the method in much more stable way

Reply:
I have separated the conditions in the updated PR.